src/local/index.js

```diff
@@ -323,6 +349,10 @@ class Local implements Side {
 
   // Delete a file or folder from the local filesystem
   destroy (doc, callback) {
+    if (inRemoteTrash(doc)) {
+      callback(null)
+      return
+    }
```

Comment:
> Can you explain this case?

When a folder is restored, since restoring is actually a move from the trash to the old location, the old folder will be destroyed.

Which makes me think I got it wrong since it'll actually be trashed, not destroyed...
So maybe this check should be moved to the `trash()` method.

> I have the impression that we should try to remove the file (and just don't throw an error if the file doesn't
> exist). For example, while cozy-desktop is uploading a big file, the user moves to the trash another file, and just
> after delete it permanently, this other file won't be deleted on the local disk after the upload finished.

Makes sense, I'll fix it.